fix: use correct fill value for parking icons

reidzeibel opened the pull request:
Relates to https://github.com/AtB-AS/kundevendt/issues/15241

### Problem
Current parking icon SVG doesn't allow us to apply fill in the app. It won't take the fill value the `fill` parameter on the SVG file is specified as `none`.

Example from the assets generated inside the mittatb-app repo (src/assets/svg/color/icons/vehicles/light/Parking.tsx):

![image](https://github.com/AtB-AS/design-system/assets/1777333/8c35797a-6ada-40ea-8044-0023ca95d31e)

Red highlights the wrong behavior, blue highlights the correct behavior, the blue highlighted icon is styled using `fill` value on the app and is working as intended.

![image](https://github.com/AtB-AS/design-system/assets/1777333/6ecbb062-4157-4f3c-bd2e-541846c8be25)

### Solution

Add correct asset to `mono` folder instead of using one in `color` icon, maybe it's related to how the asset is generated in each folders.


### Old solution
~~removing the `fill` parameter inside the `<Path>` tag, and instead move the value of that parameter to the `<Svg>` tag instead.~~

~~instead of:~~

```
<Svg fill="none">
   <Path fill="black" d="..."/>
   <Path fill="black" d="..."/>
</Svg>
```

~~we can do:~~

```
<Svg fill="black">
   <Path d="..."/>
   <Path d="..."/>
</Svg>
```
~~The resulting SVG will still look the same as previous, but now we can override the fill value on the app (or web).~~

~~There are some other files that have `fill="none"` value, but it should be changed only when there are issues, to limit the scope of checking/verification in the PR.~~

gorandalum commented:
It might be something with the export from Figma, we don't normally edit these "by hand". Maybe have a quick chat with the designers about how to export these.

reidzeibel commented:
@gorandalum I reverted the previous change and added a new file to correct location (mono icons instead of color icons), @rosvik thinks that it might have something to do with the asset generation from `mono` and `color` folders.

rosvik reviewed (approved):
LGTM 👍
